vercel.json

```diff
@@ -445,10 +445,20 @@
       "permanent": false
     },
     {
-      "source": "/docs/guides/console/create-project",
+      "source": "/docs/concepts/project",
+      "destination": "https://www.ory.sh/docs/welcome",
+      "permanent": false
+    },
+    {
+      "source": "/docs/concepts/terminology",
       "destination": "https://www.ory.sh/docs/welcome",
       "permanent": false
     },
+    {
+      "source": "/docs/guides/console/create-project",
+      "destination": "https://www.ory.sh/docs/guides/console/create-project",
```

Comment:
Bruh you're now redirecting `https://www.ory.sh/docs/guides/console/create-project` to `https://www.ory.sh/docs/guides/console/create-project` ...